client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PopoverMenu from 'components/popover/menu';
+import PopoverMenuItem from 'components/popover/menu-item';
+import Gridicon from 'components/gridicon';
+
+class TaxonomyManagerListItem extends Component {
+	static propTypes = {
+		name: PropTypes.string,
+		translate: PropTypes.func,
+		onClick: PropTypes.func,
+	};
+
+	static defaultProps = {
+		onClick: () => {}
+	};
+
+	constructor( props ) {
+		super( props );
+		this.state = {
+			popoverMenuOpen: false
+		};
+	}
+
+	togglePopoverMenu = () => {
+		this.setState( {
+			popoverMenuOpen: ! this.state.popoverMenuOpen
+		} )
+	}
```

Comment:
>I think it's probably better to add a semicolon here like class properties above, but I think we don't have any ESLint rule about this.

I've seen a tendency toward adding the semicolon too. I'll see if an updated version of ESLint detects these better.